inst/stan/epinowcast.stan

```diff
@@ -125,8 +125,9 @@ transformed parameters{
 model {
   // priors for unobserved expected reported cases
   leobs_init ~ normal(eobs_init, 1);
+  eobs_lsd ~ normal(eobs_lsd_p[1], eobs_lsd_p[2]);
+  target += -normal_lccdf(0 | eobs_lsd_p[1], eobs_lsd_p[2]);
```

Comment:
Did you maybe forget to multiply the `-normal_lccdf` part with the number of groups `g`? I think the complementary cdf must be subtracted once for each "observation".